shelley/chain-and-ledger/formal-spec/updates.tex

```diff
@@ -0,0 +1,22 @@
+\section{Software Updates}
+\label{sec:updates}
+
+This section describes how software updates are performed.
+The process of upgrading the system a new version consists of:
+
+\begin{enumerate}
+  \item New software is ready for downloading. The core nodes propose changing the 
+    \textit{software version} in the protocol parameters. If accepted, the proposal
+    will change the protocol parameters at the start of the next epoch.
+  \item All nodes see a new software version and (hopefully) update their software.
+    Note that the new software will be compatible with the older version, and no changes to
```

Comment:
Is there something that prevents the new software from being incompatible with the old version? If not maybe we should rephrase this as saying "it should be compatible with the old version" :man_shrugging:

Reply:
hmm, ok, that's fair. kinda like my "hopefully" above :)